modules/ROOT/pages/index.adoc

```diff
@@ -1,27 +1,40 @@
-= An Example Quick Course
+= Getting Started with Red Hat Connectivity Link
 :navtitle: Home
 
 == Introduction
 
-Course Title: Course title FIXME
+Welcome to this quick course on the _Getting Started with Red Hat Connectivity Link_.
+This course is the `second` in a series of `two` courses about Red Hat Connectivity Link.
```

Comment:
List both names in order here with links.

Reply:
added the link